features/step_definitions/pseud_steps.rb

```diff
@@ -19,3 +25,22 @@
   visit user_pseuds_path(User.current_user)
   click_link("delete_#{pseud}")
 end
+
+When /^"([^\"]*)" creates the default pseud "([^"]*)"$/ do |username, newpseud|
+  visit new_user_pseud_path(username)
+  fill_in "Name", with: newpseud
+  check("pseud_is_default")
+  click_button "Create"
+end
+
+When /^"([^"]*)" creates the pseud "([^"]*)"$/ do |username, newpseud|
+  visit new_user_pseud_path(username)
+  fill_in "Name", with: newpseud
+  click_button "Create"
+end
+
+When /^I create the pseud "([^"]*)"$/ do |newpseud|
```

Comment:
This step is identical to:

https://github.com/otwcode/otwarchive/blob/09565b8ee6f850fe22eab08823940b7347b849bb/features/step_definitions/pseud_steps.rb#L12-L16

We can remove it and use the other one.

Reply:
Oops, yeah, I updated the step that relied on this one, but forgot to delete it when I was done. 🤦‍♂